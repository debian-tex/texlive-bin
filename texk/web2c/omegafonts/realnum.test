#! /bin/sh

# Copyright (C) 2012 Peter Breitenlohner <tex-live@tug.org>
# You may freely use, modify and/or distribute this file.

test -d tests || mkdir -p tests

TEXMFCNF=$srcdir/../../kpathsea
OFMFONTS=tests
export TEXMFCNF OFMFONTS

./omfonts -opl2ofm -verbose $srcdir/tests/realnum tests/xrealnum || exit 1

./omfonts -ofm2opl -verbose xrealnum tests/xrealnum || exit 1

diff $srcdir/tests/realnum.out tests/xrealnum.opl || exit 1

