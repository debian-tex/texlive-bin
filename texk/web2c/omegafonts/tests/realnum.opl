(DESIGNSIZE R 10.0)
(CHARACTER C A
   (COMMENT the R must be followed by a space, tab, or sign)
   (CHARWD R )
   (CHARHT R 1.0)
   (CHARDP R 0.1)
   (CHARIC R+1)
   )
(CHARACTER C B
   (CHARWD R .)
   (CHARHT R 1.)
   (CHARDP R .1)
   (CHARIC R--1)
   )
