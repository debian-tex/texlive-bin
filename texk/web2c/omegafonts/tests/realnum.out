(FAMILY UNSPECIFIED)
(FACE F MRR)
(CODINGSCHEME UNSPECIFIED)
(DESIGNSIZE R 10.0)
(COMMENT DESIGNSIZE IS IN POINTS)
(COMMENT OTHER SIZES ARE MULTIPLES OF DESIGNSIZE)
(CHECKSUM H 0)
(FONTDIMEN
   )
(COMMENT
   (CHARWD_ENTRY D 0 R 0.0)
   (CHARWD_ENTRY D 1 R 0.0)
   )
(COMMENT
   (CHARHT_ENTRY D 0 R 0.0)
   (CHARHT_ENTRY D 1 R 1.0)
   )
(COMMENT
   (CHARDP_ENTRY D 0 R 0.0)
   (CHARDP_ENTRY D 1 R 0.1)
   )
(COMMENT
   (CHARIC_ENTRY D 0 R 0.0)
   (CHARIC_ENTRY D 1 R 1.0)
   )
(CHARACTER H 41
   (CHARWD R 0.0)
   (CHARHT R 1.0)
   (CHARDP R 0.1)
   (CHARIC R 1.0)
   )
(CHARACTER H 42
   (CHARWD R 0.0)
   (CHARHT R 1.0)
   (CHARDP R 0.1)
   (CHARIC R 1.0)
   )
